utils.py

```diff
@@ -272,7 +272,12 @@ def convert_png_binary_to_data_url(content):
 
 
 def convert_png_to_data_url(filepath):
-    """Converts the png file at filepath to a data URL."""
+    """Converts the png file at filepath to a data URL.
+    Arg:
```

Comment:
Ditto

Reply:
@kevinlee12 , hello i'm sorry i mistakenly committed main.py file with it, is there way i can deattach from here?? On basis of your comment actually i'm not sure here what you are asking me to do, please can you explore more it'll be a great help
Thank you

Reply:
See my comment above.

Reply:
Done